StRoot/Sti/StiHit.cxx

```diff
@@ -147,15 +148,20 @@ static int nCall =0; nCall++;
       mz = gz;
       double dd[3]={mx,my,mz};
 
-
-      if (!detector->insideL(dd,7,1.2)) {
+//		Special TPC case. Pileup tracks could be very far
+//              from volume in Z for splitted sectors
+      int mask = 7;
+      if (detector->getGroupId()==kTpcId) mask = 3;
+      if (!detector->insideL(dd,mask,1.2)) {
          LOG_ERROR <<
            Form("**** StiHit.%s outside (%d) by %g (%g) ****"
            ,detector->getName().c_str()
 	   ,StiDetector::mgIndex
 	   ,StiDetector::mgValue[0],StiDetector::mgValue[1])
          << endm;
-          assert( detector->insideL(dd,7,1.5));     // 
+	 //	 assert( detector->insideL(dd,7,1.5));     // 
+	 static Int_t ibreak = 0;
+	 ibreak++;
```

Comment:
...except remove the assert?

Reply:
GitHub's UI allows comments on specific lines by selecting them, so the two lines I selected do not remove the assert. My suggestion is to remove the two lines with the `ibreak` variable.

```suggestion
```

Regarding the assert, I think it was removed because it is not compatible with the new logic in the preceding `if` statement.

Reply:
commented out